### ISSUE TITLE
Stool microbiome and metabolome differences between colorectal cancer patients and healthy adults

### ISSUE THREAD

**Issue by Peacesandy** [waldronlab/BugSigDBcuration#288]
Stool microbiome and metabolome differences between colorectal cancer patients and healthy adults -- Tiffany L Weir et al. - PLoS One

https://pubmed.ncbi.nlm.nih.gov/23940645/

**Comment by BarakatAA** [comment]
Hi, please can this be assigned to me?

**Comment by omojokunoluwatomisin10** [comment]
can i be assigned to this @Peacesandy

**Comment by BarakatAA** [comment]
Hi @SvetlanaUP @Peacesandy, good morning

When I tried creating a new study page to curate this study, I got the message 'must be unique'
It seems the study as been curated already and exists on the bugsigdb database

**Comment by SvetlanaUP** [comment]
@BarakatAA yes, you are right, here it is https://bugsigdb.org/Study_589
However, Experiment 1 needs review, and you could actually review it and report here the changes.

**Comment by BarakatAA** [comment]
Okay @SvetlanaUP I'll get to it

**Comment by BarakatAA** [comment]
Good morning @SvetlanaUP , I have concluded the review for Experiment 1 of the study. Please find below a summary of my changes and observations:

1)Alpha diversity was not recorded for the study whereas in the Results and Discussion segment of the study the authors made mentions of findings in the following alpha diversities; Richness, chao1 and Shannon. I recorded them as follows in line with what the researchers discovered and reported in their study:
Richness – was decreased in CRC patients compared to healthy controls where it was increased
Chao1 – was also decreased in CRC patients compared to healthy controls, while
Shannon (species richness and evenness) was unchanged as there were no significant differences found between CRC patients and the control group.

2)Student’s t-test which the curator recorded as one of the statistical measures was flagged in bugsigdb because it’s not in the bugsigdb database so I replaced with its interchangeable term T-test which is in the database.

3)While linear regression was performed for abundant taxa in both samples against participants age and BMI and a negative correlation was found, the study didn’t state that participants were selected for the study based on these two criteria so I could not enter them as factors the study was matched on. Also, the ages for participants seen in table 1 ranged from 30 to 85 for CRC patients and 24-67 for healthy individuals showing no intentional age-matching was done, same as BMI.

4)While the study’s abstract lists sample size of healthy control and CRC patients as 11 and 10 respectively, Table 1 which lists out the participant IDs shows that indeed there were 11 CRC samples and 10 healthy samples as reported by the curator
4) Finally, there were other exclusion criteria other than antibiotics stated for the study which I included in a talk page.
Thank you

**Comment by Buraah** [comment]
Hi @SvetlanaUP,
Experiment 1 of this study still needs review. Should I go ahead?

**Comment by Buraah** [comment]
Hi @SvetlanaUP, I am done reviewing this study.

Here are the changes I made;

1. **Statistical Test:** The study used two statistical tests, ANOVA and T-test, but only T-test was used to determine differential abundance. I removed ANOVA from the curation and added Raw Counts as the data transformation for T-test.

2. **Sequencing Platform:** The sequencing platform was not specified or mentioned in the study and supplemental materials. I removed "Roche454" and left it blank.

3. Group name & Definition: I edited these to make them more descriptive.

Thank you.
